client/src/globals.js

```diff
@@ -1,6 +1,9 @@
 import { Colors } from "@blueprintjs/core";
 import { dispatchNetworkErrorMessageToUser } from "./util/actionHelpers";
 
+/* flags for testing */
+export const autosaveEnabled = process.env.AUTOSAVE_ENABLED ? process.env.AUTOSAVE_ENABLED === "true" : true;
```

Comment:
One last-minute thought:  I think we will need to remove this at some point, as I can't see a world where we (literally) don't save a user's annotations.

Can you give it a name more suggestive of its temporary "work around" nature?

Eg, "tmpWorkAroundAutosaveEnabled"?

Reply:
I'm just going to implement the `/tmp` file idea and push it to this PR here in a bit.